ci/util/serial_interface.py

```
@@ -214,12 +214,17 @@ def _producer() -> None:
+        serial output (rebooted and running), avoiding fixed sleep delays.
+        """
         if not hasattr(self._monitor, "reset_device"):
-            # Older fbuild version — fall back to pyserial
             return _pyserial_dtr_reset(self._monitor.port)
         try:
-            result = await self._run_in_thread(self._monitor.reset_device, board)
+            result = await self._run_in_thread(
+                self._monitor.reset_device, board, True, 5.0
+            )
             return bool(result)
         except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)